pydicom/charset.py

```diff
@@ -95,6 +95,45 @@
                      'iso_ir_58')
 
 
+# shift_jis is a superset of jis_x_0201. So we can regard the encoded value
```

Comment:
Could you make that a docstring to be consistent with the other functions? Same for the other functions.

Reply:
I added docstrings to new functions I had added.

Reply:
Very nice!